packages/nimble-components/src/toggle-button/styles.ts

```diff
@@ -1,20 +1,34 @@
 import { css } from '@microsoft/fast-element';
-import { focusVisible } from '../utilities/style/focus';
 
 import {
-    borderHoverColor,
-    borderWidth,
+    buttonAccentOutlineFontColor,
+    buttonLabelFontColor,
     fillSelectedColor,
-    fillSelectedRgbPartialColor
+    fillSelectedRgbPartialColor,
+    iconColor
 } from '../theme-provider/design-tokens';
-import { styles as buttonStyles } from '../patterns/button/styles';
+import {
+    buttonAppearanceVariantStyles,
+    styles as buttonStyles
+} from '../patterns/button/styles';
+import { appearanceBehavior } from '../utilities/style/appearance';
+import { ButtonAppearance } from './types';
 
 export const styles = css`
+    @layer base, base-selected, hover, focusVisible, active, disabled, top;
```

Comment:
By inserting a new `base-selected` layer, I can define the default styling for a toggled-on button, but still let standard hover, focus, active, and disabled styling take precedence.

Reply:
if we do need the new layer "selected" to me was ambiguous with "active" / "focus" which are the current "user selected" thing to interact with.

This comment makes me think more "pressed" going with the aria naming. And I'd leave off the 'base' prefix.

Reply:
Unfortunately, I think "pressed" is ambiguous with "active", since a button is "active" while it is pressed. I suggest that we instead call it "checked", since that term matches the classes and attribute applied in that state. Plus, if we later introduce similar layers for other controls with binary state, that name will make sense.